### ISSUE TITLE
Optimize consistency between the studies and discoveries section in the app

### ISSUE THREAD

**Issue by AMelhede** [curedao/curedao-monorepo#43]
The relationship between a given variable and the actual figure is usually inconsistent.

Example below:

![image](https://user-images.githubusercontent.com/95473109/226610997-95706e65-5cbf-435e-b1be-1e616aa962b1.png)

Also, the guiltiness in the example is not found in the study, which is linked to:
https://app.quantimo.do/variables/1872

More examples of the relationships below:
![image](https://user-images.githubusercontent.com/95473109/226612131-fa0704e7-0eb7-4fcc-be3e-a0af3b38f6be.png)

![image](https://user-images.githubusercontent.com/95473109/226612070-562c6335-677b-4aa7-8d55-d5fb38d28c73.png)

Link to the vitamin D mega study mentioned in the example above:
https://app.quantimo.do/variables/95802

Also, the studies show similar descriptions, such as "very slightly higher" ,"very slightly lower" for huge deviations in percentage:

![image-2](https://user-images.githubusercontent.com/95473109/228464387-90692454-b8b0-4b1c-8f55-938b3ac8606c.png)
![image-1](https://user-images.githubusercontent.com/95473109/228464393-5c68ed1a-4943-4f28-b6e9-799517afdb1c.png)
![image](https://user-images.githubusercontent.com/95473109/228464396-0f432707-519e-4eb9-9cc0-42b85ee298c3.png)

**Comment by mikepsinn** [comment]
Good point. That's a difference between the method of analysis used to calculate the change from baseline and the predictive relationship strength. When there is sparse data and the effect is small, these different methods can produce different results. So we'll need to just derive the title from the baseline calculation, as it's more reliable and doesn't require a normal distribution.

**Comment by AMelhede** [comment]
Okay, that sounds great. Thank you for explaining that, @mikepsinn.
I imagine many would find it misleading to see so many different results for the same intervention.
Fixing this would indeed be very helpful to add credibility and precision of the studies long-term.
